content/en/serverless/guide/azure_app_service_linux_sidecar.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Instrument Azure App Service - Linux Containers with severless-init
+kind: guide
+further_reading:
+- link: "/integrations/azure_app_services/"
+  tag: "Documentation"
+  text: "Azure App Service"
+- link: "/integrations/azure_app_service_environment/"
+  tag: "Documentation"
+  text: "Azure App Service Environment"
+---
+
+<div class="alert alert-info">To instrument your Azure App Service containers with <code>serverless-init</code>, see <a href="/serverless/azure_app_services_container">Instrument Azure App Service - Linux Container with serverless-init</a>.</div>
+
+## Overview
+
+This instrumentation method uses Azure's [Sidecar pattern][1] to monitor containerized Linux Azure App Service workloads.
+
+### Prerequisites
+
+- Your Azure App Service application is containerized
+- You are using a programming language [supported by a Datadog tracing library][2]
+- You have a [Datadog API key][3]
+
+## Instrument your application
+
+1. [Integrate the Datadog tracer][11] into your containerized application
+1. [Create your Linux Web App][12]
+1. [Add Datadog environment variables][13] as application settings
+1. [Add the Datadog sidecar][14]
+
+### Integrate the Datadog tracer
+
+After your application is containerized, integrate the Datadog tracer by adding the following lines to the Dockerfile for your main application.
+
+{{< programming-lang-wrapper langs="dotnet" >}}
+{{< programming-lang lang="dotnet" >}}
+
+```dockerfile
+RUN mkdir -p /datadog/tracer
+RUN mkdir -p /home/LogFiles/dotnet
+
+ADD https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.51.0/datadog-dotnet-apm-2.49.0.tar.gz /datadog/tracer
+RUN cd /datadog/tracer && tar -zxf datadog-dotnet-apm-2.49.0.tar.gz
+```
+
+This installs and configures the Datadog tracer within your application container.
+
+#### Full example Dockerfile
+
+```dockerfile
+# Stage 1: Build the application
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+WORKDIR /app
+
+# Copy the project file and restore dependencies
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy the remaining source code
+COPY . .
+
+# Build the application
+RUN dotnet publish -c Release -o out
+
+# Stage 2: Create a runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS runtime
+WORKDIR /app
+
+# Copy the build output from stage 1
+COPY --from=build /app/out ./
+
+# Datadog specific
+RUN mkdir -p /datadog/tracer
+RUN mkdir -p /home/LogFiles/dotnet
+
+ADD https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.49.0/datadog-dotnet-apm-2.49.0.tar.gz /datadog/tracer
+RUN cd /datadog/tracer && tar -zxf datadog-dotnet-apm-2.49.0.tar.gz
+
+# Set the entry point for the application
+ENTRYPOINT ["dotnet", "<your dotnet app>.dll"]
+```
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+Then, build the image and push it to your preferred container registry.
```

Comment:
This is getting lost between all the big code samples. Two suggestions: first, make this an ordered list even though it's only 2 steps. Second, move this line above the full Dockerfile example.